static/src/javascripts/projects/common/modules/loyalty/save-for-later.js

```diff
@@ -21,14 +21,15 @@ define([
         this.pageId = config.page.pageId;
         this.$saver = bonzo(this.saveLinkHolder);
         this.savedArticlesUrl = config.page.idUrl + '/saved-articles';
+        this.shortUrl = config.page.shortUrl.replace('http:/', '');   //  Keep the fitst trailing slash
```

Comment:
Shouldn't this remove the /gu.com/ bit too? I thought we were just seeing /p/kj2khj4 in identity?